enhancements/kubelet/user-namespaces-support.md

```diff
@@ -0,0 +1,454 @@
+| Groups                          | N/A                         | No impact--this field is dictating which openshift user can access this, and is not related to a pod field.                                                                                                               |
+| SeccompProfiles                 | seccompProfile              | No impact if a pod is in a user namespace on this field, but if this field is set to `*` or "empty" then it may be tempting to allow all capabilities, but the privileged escalation piece is another aspect to consider. |
+| AllowedUnsafeSysctls            | sysctls                     | No impact--the handling of sysctls should not change if a pod is in a user namespace.                                                                                                                                     |
+| ForbiddenSysctls                | sysctls                     | No impact--the handling of sysctls should not change if a pod is in a user namespace.                                                                                                                                     |
+
+##### `UserNamespaceLevel` field
+
+Add the following to the [Openshift API](https://github.com/openshift/api/blob/76a71dac36a08eab1b240c6c8d4e39c813b1b12b/security/v1/types.go):
+
+```
+       // userNamespaceLevel determines if the policy allows host users in containers.
+       // Valid values are "AllowHostLevel", "RequirePodLevel", and omitted.
+       // When "AllowHostLevel" is set, a pod author may set `hostUsers` to either `true` or `false`.
+       // When "RequirePodLevel" is set, a pod author must `hostUsers` to `false`.
+       // When omitted, this means no opinion and the platform is left to choose a reasonable default,
+       // which is subject to change over time.
+       // The current default is "AllowHostLevel".
+       // +openshift:enable:FeatureGate=UserNamespacesPodSecurityStandards
+       // +kubebuilder:validation:Enum="AllowHostLevel";"RequirePodLevel";""
+       // +optional
+       UserNamespaceLevel NamespaceLevelType `json:"userNamespaceLevel" protobuf:"bytes,26,opt,name=userNamespaceLevel"`
+...
+// NamespaceLevelType shows the allowable values for the UserNamespaceLevel field.
+type NamespaceLevelType string
+...
+const (
+       // NamespaceLevelAllowHost allows a pod to set `hostUsers` field to either `true` or `false`
+       NamespaceLevelAllowHost NamespaceLevelType = "AllowHostLevel"
+       // NamespaceLevelRequirePod requires the `hostUsers` field be `false` in a pod.
+       NamespaceLevelRequirePod NamespaceLevelType = "RequirePodLevel"
+       // NamespaceLevelEmpty will set to the default, which is currently "AllowHostLevel".
+       NamespaceLevelEmpty NamespaceLevelType = ""
```

Comment:
This field is closer to user workloads than to configuration. This one we'll default to `AllowHostLevel` instead of leaving empty because we know the intent will never change

Reply:
sorry I don't think I fully understand--do you want me to drop this or keep it?

Reply:
Empty should not be allowed.  We should default the value.

Reply:
I dropped empty.

> Is the expectation there that we create a new version of the SCC and folks move over to it?

this is my understanding of the approach